public/resources/remix/index.js

```diff
@@ -21,36 +20,37 @@
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
     var detailsBar = document.querySelector(".details-bar");
-    detailsBar.setAttribute("style", "");
+    
+    if (detailsBar) {
+      detailsBar.setAttribute("style", "");
+      if (isTouchDevice) {
```

Comment:
we might need to consider what we do here a bit more:  since we add additional functionality based on `mouse-mode`, what we *want* to know is "is there a mouse cursor or not", so we should figure out how to test for that. Just because something is a touch device does not mean there won't also be a mouse cursor available.

(My tablet, for instance, has a dock with a mouse pad on it. Despite being a touch device, mouseenter and mouseleave will work just fine)

thoughts @flukeout, @humphd?